### ISSUE TITLE
dear teaachers, could you please tell me? How long does the distillation code in the following configuration take on an 80G A100, thanks!

### ISSUE THREAD

**Issue by leo23ui** [microsoft/Cream#244]
export NNODES=1
export GPUS_PER_NODE=1

DISTRIBUTED_ARGS="--nproc_per_node $GPUS_PER_NODE --nnodes $NNODES"
torchrun $DISTRIBUTED_ARGS src/training/main.py \
 --save-frequency 1 \
 --report-to wandb \
 --train-data /home/gg/gg/MQBench-main/test/model/c/yfcc15m.tar \
 --dataset-type webdataset \
 --imagenet-val ./ImageNet \
 --warmup 2000 \
 --batch-size 512 \
 --epochs 25 \
 --workers 1 \
 --model TinyCLIP-ViT-39M-16-Text-19M \
 --name exp_name \
 --seed 0 \
 --local-loss \
 --grad-checkpointing \
 --output ./outputs/TinyCLIP-ViT-39M-16-Text-19M \
 --lr 0.0001 \
 --gather-with-grad \
 --pretrained-image-file ViT-B-16@openai \
 --pretrained-text-file ViT-B-16@openai \
 --distillation-teacher ViT-B-32@laion2b_e16 \
 --norm_gradient_clip 5 \
 --train-num-samples 15000000 \
 --logit-scale 50

**Comment by wkcn** [comment]
Hi @leo23ui , thanks for your attention to our work!

We used 16 V100 GPUs to train `TinyCLIP-ViT-39M-16-Text-19M` on YFCC-15M for 25 epochs with the distillation teacher.
It took 35 hours, where the throughput is 3,842 pairs per second.

**Comment by leo23ui** [comment]
> Hi [@leo23ui](https://github.com/leo23ui) , thanks for your attention to our work!
> 
> We used 16 V100 GPUs to train `TinyCLIP-ViT-39M-16-Text-19M` on YFCC-15M for 25 epochs with the distillation teacher. It took 35 hours, where the throughput is 3,842 pairs per second.

thanks for your reply!! have a good day!!
